react/src/js/components/Consonant/CardsCarousel/CardsCarousel.jsx

```diff
@@ -115,63 +176,80 @@ function CardsCarousel({
         }
     }
 
-    function mouseDownHandler(e) {
-        e.preventDefault();
-        interactedWith = true;
-        responsiveLogic();
-        isDown = true;
-        startX = e.pageX;
+    /**
+     * 620 = (tablet range) + average grid gap
+     * 620 = 1200px - 600px + (8 + 32)/2
+     */
+    function centerClick() {
+        const carousel = carouselRef.current;
+        /* eslint-disable-next-line no-mixed-operators */
+        carousel.scrollLeft += (-window.innerWidth / 2 + 620);
     }
 
-    function mouseUpHandler() {
-        isDown = false;
-        isMouseMove = false;
-    }
+    /**
+     * Jira ticket: MWPW-164509
+     * Sets the ARIA attributes for the cards based on their visibility.
+     * @param {HTMLElement} carousel - The carousel element.
+     */
+    function setAriaAttributes(carousel) {
+        const shouldRenderOverlay = renderOverlay || cardStyle === 'half-height';
 
-    function mouseLeaveHandler() {
-        isDown = false;
-        isMouseMove = false;
-    }
+        carousel.querySelectorAll('.consonant-Card').forEach((card, index) => {
+            const cardLink = shouldRenderOverlay
+                ? card.querySelector('.consonant-LinkBlocker')
+                : card.querySelector('.consonant-BtnInfobit');
 
-    function mouseMoveHandler(e) {
-        if (!isDown) return;
-        isMouseMove = true;
-        const carousel = carouselRef.current;
-        const x = e.pageX - carousel.offsetLeft;
-        carousel.scrollLeft -= (x - startX);
-    }
+            if (!cardLink) return;
 
-    function scrollHandler() {
-        interactedWith = true;
-        responsiveLogic();
+            if (index + 1 >= firstVisibleCard && index + 1 <= lastVisibleCard) {
+                cardLink.removeAttribute('aria-hidden');
+                cardLink.removeAttribute('inert');
+                cardLink.setAttribute('tabindex', '0');
+            } else {
+                cardLink.setAttribute('aria-hidden', 'true');
+                cardLink.setAttribute('inert', '');
```

Comment:
HTML `inert` attribute is not universally supported (e.g. Safari). Do we need to ship a polyfill as well?